core/poi/lib/plugins/config-css.js

```diff
@@ -64,6 +64,10 @@ exports.apply = api => {
             rule
               .use('extract-css-loader')
               .loader(require('mini-css-extract-plugin').loader)
+              .options({
+                hmr: api.mode === 'development',
+                reloadAll: true
+              })
```

Comment:
I just realized that we need to enable css extraction for non-production mode as well.

Reply:
Oops, so this feature is not practical.

Reply:
just make `css.extract` default to `true` in development mode as well.

Reply:
But, is css extract useful in development mode?